setup.py

```diff
@@ -0,0 +1,50 @@
+"""
+Export Directory API client
+"""
+import ast
+import pip.download
+from pip.req import parse_requirements
+import re
+from setuptools import setup, find_packages
+
+
+def get_version():
+    pattern = re.compile(r'__version__\s+=\s+(.*)')
+
+    with open('directory_forms_api_client/version.py', 'rb') as src:
+        return str(ast.literal_eval(
+            pattern.search(src.read().decode('utf-8')).group(1)
+        ))
+
+
+def get_requirements():
+    return [str(r.req) for r in list(parse_requirements(
+        'requirements.txt',
+        session=pip.download.PipSession()
```

Comment:
doesn't work with newer versions of pip - see sigauth - we might need to hardcode the reqs or find a new method of loading from the txt

Reply:
this is true for all of our libs. lets do a spike to find a way to solve.